ckanext/data_qld/plugin.py

```diff
@@ -1,268 +0,0 @@
-    plugins.implements(plugins.ITemplateHelpers)
-    plugins.implements(plugins.IValidators)
-    plugins.implements(plugins.IPackageController, inherit=True)
-    plugins.implements(plugins.IAuthFunctions)
-    plugins.implements(plugins.IActions)
-    plugins.implements(plugins.IRoutes, inherit=True)
-    plugins.implements(plugins.IResourceController, inherit=True)
-    plugins.implements(plugins.IMiddleware, inherit=True)
-    plugins.implements(plugins.IBlueprint)
-
-    if ' qa' in toolkit.config.get('ckan.plugins', ''):
-        plugins.implements(IQA)
-
-    # IConfigurer
-    def update_config(self, config_):
-        toolkit.add_template_directory(config_, 'templates')
-        toolkit.add_public_directory(config_, 'public')
-        toolkit.add_resource('fanstatic', 'data_qld')
-
-    def update_config_schema(self, schema):
-        ignore_missing = toolkit.get_validator('ignore_missing')
-        schema.update({
-            # This is a custom configuration option
-            'ckanext.data_qld.datarequest_suggested_description': [ignore_missing, unicode],
-            'ckanext.data_qld.resource_formats': [ignore_missing, unicode]
-        })
-        return schema
-
-    # ITemplateHelpers
-    def get_helpers(self):
-        return {'data_qld_data_driven_application': helpers.data_driven_application,
-                'data_qld_dataset_data_driven_application': helpers.dataset_data_driven_application,
-                'data_qld_datarequest_default_organisation_id': helpers.datarequest_default_organisation_id,
-                'data_qld_organisation_list': helpers.organisation_list,
-                'data_qld_datarequest_suggested_description': helpers.datarequest_suggested_description,
-                'data_qld_user_has_admin_access': helpers.user_has_admin_access,
-                'data_qld_format_activity_data': helpers.format_activity_data,
-                'data_qld_resource_formats': helpers.resource_formats,
-                'activity_type_nice': helpers.activity_type_nice,
-                'profanity_checking_enabled': helpers.profanity_checking_enabled
-                }
-
-    # IValidators
-    def get_validators(self):
-        return {
-            'data_qld_filesize_converter': converters.filesize_converter,
-            'data_qld_filesize_formatter': converters.filesize_formatter,
-            'data_qld_scheming_choices': validation.scheming_choices,
-        }
-
-    # IPackageController
-    def set_maintainer_from_author(self, entity):
-        entity.author = entity.author_email
-        entity.maintainer = entity.author_email
-        entity.maintainer_email = entity.author_email
-
-    def create(self, entity):
-        self.set_maintainer_from_author(entity)
-
-    def edit(self, entity):
-        self.set_maintainer_from_author(entity)
-
-    # IAuthFunctions
-    def get_auth_functions(self):
-        auth_functions = {
-            constants.UPDATE_DATAREQUEST: datareq_auth.update_datarequest,
```

Comment:
I thought we did customisation of datarequest plugin here?

Reply:
That's all in `integration_plugin.py` now.